tests/testthat/test-assertions.R

```diff
@@ -150,3 +150,9 @@ test_that("assert_order_vars Test 13: returns errors if used incorrectly", {
   expect_error(assert_order_vars(c("USUBJID", "PARAMCD", "VISIT")))
   expect_error(assert_order_vars(vars(USUBJID, toupper(PARAMCD), -AVAL)))
 })
+
+# assert_date_vector ----
+## Test 14: returns error if input vector is not a date formatted ----
+test_that("assert_date_vector Test 14: returns error if input vector is not a date formatted", {
+  expect_error(assert_date_vector("2018-08-23"))
```

Comment:
Please also add a test case where no error is thrown because the input is valid. In general `expect_error(some_fun(), NA)` will check that **no** error is thrown.

Reply:
update implemented